content/docs/2_cookbook/9_setup/0_caddy/cookbook-recipe.txt

```diff
@@ -0,0 +1,124 @@
+Title: Running Kirby on a Caddy v2 web server
+
+----
+
+Published: 2023-08-03
+
+----
+
+Description: How to configure the popular and simple Caddy v2 web server to run a Kirby website.
+
+----
+
+Authors:
+
+- authors/moritz-stueckler
+
+----
+
+Text:
+
+[Kirby states](https://getkirby.com/docs/guide/quickstart) that it is able to run on many different web servers. However, in reality it is mostly Apache and sometimes Nginx which are being used as webservers to run Kirby on. But there is also another, modern and very easy-to-configure alternative in the webserver world: [Caddy v2](https://caddyserver.com/v2).
+
+## Caddy: No support for `.htaccess`
+
+Just like Nginx the Caddy webserver does not use the famous and project-specific `.htaccess` files that Kirby ships with. So instead of using this file, you'll have to configure the webserver manually through a special configuration file called `Caddyfile`. This file should typically be placed next to the Caddy executable, or its location can be passed in, when Caddy is called.
```

Comment:
It's not the binary directory by default, but the current working directory:

```suggestion
Just like Nginx the Caddy webserver does not use the famous and project-specific `.htaccess` files that Kirby ships with. So instead of using this file, you'll have to configure the webserver manually through a special configuration file called `Caddyfile`. Caddy tries to load this file from the current directory, so run Caddy from the same folder that contains your `Caddyfile` or pass the path manually with `caddy run --config /path/to/Caddyfile`.
```